tests/ClientTest.php

```diff
@@ -212,6 +211,121 @@ public function join($timeout = null)
 
 class Raven_Tests_ClientTest extends \PHPUnit_Framework_TestCase
 {
+    protected static $_folder = null;
+
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+        self::$_folder = sys_get_temp_dir().'/sentry_server_'.microtime(true);
+        mkdir(self::$_folder);
+
+        // Root CA #A1
+        // Сертификат CA #A2, подписанный CA #A1 (с возможностью подписи)
```

Comment:
Ooops. These lines mean "Certificate CA A2, signed by CA A1 (intermediate)...". I need to remove these lines. This is copypast from my project